src/compute/src/render/reduce.rs

```diff
@@ -63,11 +63,18 @@ where
     T: Timestamp + Lattice,
 {
     // Convenience wrapper to render the right kind of hierarchical plan.
-    let build_hierarchical = |collection: Collection<G, (Row, Row), Diff>,
-                              expr: HierarchicalPlan| match expr {
-        HierarchicalPlan::Monotonic(expr) => build_monotonic(debug_name, collection, expr),
-        HierarchicalPlan::Bucketed(expr) => build_bucketed(debug_name, collection, expr),
-    };
+    let mut err_input = err_input;
```

Comment:
Nit: You don't need to re-assign to declare this variable `mut`. You can add the `mut` directly in the function parameter definition.

Alternatively, since we are now not only reading from this collection but also adding to it, you could rename the variable accordingly here, e.g.:
```
let mut err_collection = err_input;
```

Reply:
Cool, I like the renaming suggestion, so I changed to that now!